### PR TITLE
Fixup Datashader config

### PR DESCRIPTION
**Pull request by esc**
Datashader needs some logic on the repo to determine what tag is the
latest release.

### CODE REVIEW

**Comment by esc** [comment]
Title was updated to reflect that more changes to the datashader config have to be made.

**Comment by esc** [comment]
@philippjfr I'm not sure about this one, the code to determine the correct version seems unnecessarily complex.

**Comment by philippjfr** [comment]
>I'm not sure about this one, the code to determine the correct version seems unnecessarily complex.

Agreed let's just delete the non-conforming tags, i.e. website and website-dev, that should solve it automatically right?

**Comment by philippjfr** [comment]
I've deleted them now.

**Comment by esc** [comment]
I think that will not suffice, unfortunately. We still need to deal with parsing and comparing the major, minor, patch and alpha version numbers and sorting accordingly. Maybe https://pypi.org/project/packaging/ can do this for us? I'd have to check it out.

**Comment by esc** [comment]
So I tried using `packaging` it seems to work quite well for this case.

**Comment by esc** [comment]
Some of the Datashader failures are related to this issue in Numba:

https://github.com/numba/numba/issues/5570

**Comment by esc** [comment]
oh wow, datashader passed! Let's get this in!

File: switchboard.py [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python
 
 import os
+import re
```
Comment (stuartarchibald) [inline switchboard.py]:
```suggestion
```
unused import?

Comment (esc) [reply switchboard.py]: Indeed, leftover from a previous attempt at sorting/parsing version numbers.

**Review by stuartarchibald — APPROVED** [review]
Thanks for the fix, looks good.

**Pull request merged**